projects/packages/forms/src/dashboard/inbox/export-responses/google-drive.js

```diff
@@ -115,15 +113,16 @@ const GoogleDriveExport = ( { onExport } ) => {
 					) }
 
 					{ ! isConnected && (
-						<a
+						<Button
 							href={ config( 'gdriveConnectURL' ) }
 							className={ buttonClasses }
+							variant="primary"
 							rel="noopener noreferrer"
 							target="_blank"
 							onClick={ handleConnectClick }
 						>
 							{ __( 'Connect to Google Drive', 'jetpack-forms' ) }
-						</a>
+						</Button>
```

Comment:
There are some minor styling issues with the Connect to Google Drive button. Can probably be solved in a follow up. 

<img width="874" alt="google-button-styles" src="https://github.com/user-attachments/assets/b3db2c4c-8b9b-4e70-aed1-f706c8aa7e2e" />